### PR TITLE
feat(rpc): add state cmd

### PR DESCRIPTION
**Pull request by vgonkivs**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Review by ramin — APPROVED** [review]

File: cmd/celestia/state.go [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+	)
+}
+
+var stateCmd = &cobra.Command{
+	Use:   "state [command]",
+	Short: "Allows interaction with the State Module via JSON-RPC",
+	Args:  cobra.NoArgs,
+}
+
+var isStoppedCmd = &cobra.Command{
```
Comment (renaynay) [inline cmd/celestia/state.go:40]: i don't think we need this rn

File: cmd/celestia/state.go [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+			math.NewInt(amount),
+			math.NewInt(fee),
+			gasLimit,
+		)
+		return printOutput(txResponse, err, nil)
+	},
+}
+
+var delegateCmd = &cobra.Command{
+	Use:   "delegate [valAddress, amount, fee, gasLimit]",
```
Comment (renaynay) [inline cmd/celestia/state.go:340]:
```suggestion
	Use:   "delegate [valAddress] [amount] [fee] [gasLimit]",
```

applies for all cmds that take args btw

Comment (vgonkivs) [reply cmd/celestia/state.go]: fixed

**Review by walldiss — APPROVED** [review]

File: cmd/celestia/state.go (hunk 1 above)
Comment (vgonkivs) [reply cmd/celestia/state.go:40]: removed

**Comment by vgonkivs** [comment]
For some reason, lint is not working on CI(Works for me locally)

**Pull request merged**